include/swift/Basic/Statistic.h

```diff
@@ -13,7 +13,10 @@
 #ifndef SWIFT_BASIC_STATISTIC_H
 #define SWIFT_BASIC_STATISTIC_H
 
+#include "llvm/ADT/SmallString.h"
 #include "llvm/ADT/Statistic.h"
+#include "swift/Basic/Timer.h"
+#include "swift/FrontendTool/FrontendTool.h"
```

Comment:
I like that better. I think it's the best way to satisfy the letter of the layering laws for now. I wouldn't want to make more global stats anyway because of SourceKit eventually having threads.